<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>gp-counter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pics/gp_icon.ico</normaloff>pics/gp_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_drg">
       <property name="text">
        <string>Drg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_retraso">
       <property name="text">
        <string>Retraso del profeta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_fallo">
       <property name="text">
        <string>Fallo del profeta</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>19</y>
      <width>461</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="btn_back_date">
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_current_date">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_forward_date">
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>143</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="btn_amarilla">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_roja">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>50</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbl_display_info">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sin datos para esta fecha&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gp_sergio">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pics/gp_icon.ico</normaloff>pics/gp_icon.ico</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pics/players/sergio (Custom).jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pics/players/paula (Custom).jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pics/players/aina (Custom).jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pics/players/aitor (Custom).jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pics/players/anton (Custom).png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pics/players/diego (Custom).jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pics/players/joaquin (Custom).jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pics/players/laura (Custom).jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pics/players/miranda (Custom).jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pics/players/pablo (Custom).jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pics/players/nerea (Custom).jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gp_paula">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pics/gp_icon.ico</normaloff>pics/gp_icon.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gp_aina">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pics/gp_icon.ico</normaloff>pics/gp_icon.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gp_anton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pics/gp_icon.ico</normaloff>pics/gp_icon.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gp_aitor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>500</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pics/gp_icon.ico</normaloff>pics/gp_icon.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gp_diego">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pics/gp_icon.ico</normaloff>pics/gp_icon.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gp_joaquin">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pics/gp_icon.ico</normaloff>pics/gp_icon.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gp_laura">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pics/gp_icon.ico</normaloff>pics/gp_icon.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gp_miranda">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pics/gp_icon.ico</normaloff>pics/gp_icon.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_gp_nerea">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>500</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pics/gp_icon.ico</normaloff>pics/gp_icon.ico</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>550</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>gp-counter v.1.0</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="logWindow">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>381</width>
      <height>291</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_limpiar">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Herramientas</string>
    </property>
    <addaction name="actionAbrir_calendario"/>
    <addaction name="actionGenerar_reporte"/>
    <addaction name="actionLimpiar_dia"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuHerramientas"/>
  </widget>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="toolTip">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAbrir_calendario">
   <property name="text">
    <string>Abrir calendario</string>
   </property>
  </action>
  <action name="actionGenerar_reporte">
   <property name="text">
    <string>Generar reporte</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionLimpiar_dia">
   <property name="text">
    <string>Limpiar día</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
